src/entities/auth-entity.ts

```diff
@@ -3,13 +3,17 @@ import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
 @Entity()
 export class RefreshToken {
   // eslint-disable-next-line no-useless-constructor,no-empty-function
-  public constructor(token: string) {
+  public constructor(token: string, userId: string) {
```

Comment:
With which purpose there was added contructor

Reply:
to create new RefreshToken instance

Reply:
ok, then why to no remove
```js
  // eslint-disable-next-line no-useless-constructor,no-empty-function
```

Reply:
removed